sdks/python/apache_beam/yaml/yaml_provider.py

```diff
@@ -438,7 +447,7 @@ def _parse_window_spec(spec):
   return InlineProvider(
       dict({
           'Create': lambda elements,
-          reshuffle=True: beam.Create(elements, reshuffle),
+          reshuffle=True: beam.Create(dicts_to_rows(elements), reshuffle),
```

Comment:
Are we tracking work to add a docs section for these built in transforms? This is the type of behavior that would be good to document (with an example), but it doesn't seem like a structure exists for that yet.